src/Groups/GAPGroups.jl

```diff
@@ -128,7 +128,7 @@ false
 function order(::Type{T}, x::Union{GAPGroupElem, GAPGroup}) where T <: IntegerUnion
    ord = GAPWrap.Order(x.X)
    if ord === GAP.Globals.infinity
-      throw(GroupsCore.InfiniteOrder(x))
+      throw(InfiniteOrder(x))
```

Comment:
```suggestion
      throw(InfiniteOrderError(x))
```
I think you chose a different name